### ISSUE TITLE
Support for channels 2

### ISSUE THREAD

**Issue by curiouswala** [edigiacomo/asgi-mqtt#4]
When I run the asgimqtt it gives me this error
`ModuleNotFoundError: No module named 'channels.asgi'
`
Which is correct. In the current version of channels, there seem to be no channels.asgi module.
Are you planning to upgrade this to channels 2 or could give me a bit of information about how it could be done. 

If you have a fix please update the example project.

Thanks

**Comment by edigiacomo** [comment]
You're right, this project doesn't support channels 2 yet. Unfortunately, I don't have any plan to upgrade it in the near future.

If you want to create a PR, I think that the [channels documentation][1] and the [source code of daphne][2] are a good starting point.

[1]: https://channels.readthedocs.io/en/latest/index.html
[2]: https://github.com/django/daphne
